tests/api/test_api_base.py

```diff
@@ -16,4 +16,4 @@ def setUpClass(cls):
         """Class method called once before running tests in a test class."""
         cls.request_timeout = 60
         cls.assert_precision = 0.01
-        cls.client = Client(environment='sandbox',merchantNo='200043',storeNo='300014',token='0c2bac90bffefbbbf84f4b17667710fd')
+        cls.client = Client(environment='sandbox',merchantNo='200043',storeNo='300014',token='5cbfb079f15b150122261c8537086d77a')
```

Comment:
1. Why changing the token? 
2. Why put the token in here?

Reply:
1. i believe this testing token was changed a few days ago, it was giving me the error when making request
2. thats the base unit test class, just trying to initialize the client in the base file,

Reply:
@xuehao712 What's this token for? Will it changed often? Can we not hardcode this?

Reply:
its our sandbox api call credentials, it wont changed at all, i think someone changed our test credentials last week, and they changed it back

Reply:
Can we mock it if it's an unit test?